hw/ip/clkmgr/dv/cov/clkmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,464 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Generated UNR file from Synopsys UNR tool with security modules being black-boxed.
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Aug 11 14:09:51 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
```

Comment:
wait..i dont understand, these shouldn't be unreachable.. could you help me understand a bit what this is saying?

Reply:
It is coverage on a condition, and it typically refers to a combination of values for the terms. It seems in this case it says src_req == 1 and !busy == 0 is not reachable, so a src_req cannot not happen when busy?

Reply:
o wait, how do i tell it means `!busy` == 0? is it the `-1`?

Reply:
By the way, src_re_i is 0, so src_req means io_meas_ctrl_en_we is active while busy? Perhaps the tlul_adapter waits till !busy to allow a write to proceed?

Reply:
yeah that's what the adapter does. So i'm going to make a change in the RTL to remove this condition altogether and just check with an assertion.

Reply:
these should be removed with #14258 
other blocks will also have this issue, so may as well get rid of them.

Reply:
I think the individual values of the terms in this expression are encoded in the (2 "10") string. I don't know what the 1 -1 numbers mean.